omero_marshal/decode/decoders/screen.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Glencoe Software, Inc. All rights reserved.
+#
+# This software is distributed under the terms described by the LICENCE file
+# you can find at the root of the distribution bundle.
+# If the file is missing please request a copy by contacting
+# jason@glencoesoftware.com.
+#
+
+from ... import SCHEMA_VERSION
+from .annotation import AnnotatableDecoder
+from omero.model import ScreenI
+
+
+class Screen201501Decoder(AnnotatableDecoder):
+
+    TYPE = 'http://www.openmicroscopy.org/Schemas/OME/2015-01#Screen'
+
+    OMERO_CLASS = ScreenI
+
+    def decode(self, data):
+        v = super(Screen201501Decoder, self).decode(data)
+        self.set_property(v, 'name', data.get('Name'))
+        self.set_property(v, 'description', data.get('Description'))
```

Comment:
Same comment as above -there are [Screen attributes](http://www.openmicroscopy.org/Schemas/Documentation/Generated/OME-2016-06/ome_xsd.html#Screen) that should be covered in the encoder/decoder/